### PR TITLE
fix links to use hyphen

### PR DESCRIPTION
**Pull request by Lendemor**

### CODE REVIEW

File: pcweb/pages/docs/__init__.py [hunk 1]
```diff
@@ -155,14 +155,16 @@ def get_component(doc: str, title: str):
 
 doc_routes = [gallery, library, resources] + apiref_pages
 
-
+t = True
```
Comment (picklelo) [inline pcweb/pages/docs/__init__.py:158]: leftover?

**Review by picklelo — CHANGES_REQUESTED** [review]
Actually I think the move here is to modify the `.md` file names to always use `-` - so there's consistency between the names and the url. Then maybe we just add an assert here that it doesn't contain an underscore

**Review by picklelo — COMMENTED** [review]
Looks good, I think we just have to modify how we do the dynamic name spacing now (convert to snake_case before setting it in the namespace)

**Comment by Lendemor** [comment]
> Looks good, I think we just have to modify how we do the dynamic name spacing now (convert to snake_case before setting it in the namespace)

Investigating why but they are already snake_cased in the namespace, yet it still fail.

**Review by picklelo — APPROVED** [review]

**Pull request merged**